src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -243,4 +255,37 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	protected toggleMoreActionsButton(isActiveOrHovered: boolean) {
 		this._cellToggleMoreActions.toggleVisible(!isActiveOrHovered);
 	}
+
+	public deltaDecorations(newDecorationRange: NotebookRange, oldDecorationRange: NotebookRange): void {
```

Comment:
I'm not sure what the name of this method is supposed to say it does - delta usually refers to the difference between things but this just appears to be adding/removing decorations?

Reply:
I followed what vscode has. They have this deltadecorations where the remove the old ones and add the new ones.